$MirOS: ports/misc/screen/patches/patch-doc_screen_texinfo,v 1.2 2007/02/11 01:33:45 tg Exp $
--- doc/screen.texinfo.orig	Fri Dec  5 13:51:24 2003
+++ doc/screen.texinfo	Sat May 18 01:36:33 2013
@@ -13,13 +13,8 @@
 * Screen: (screen).             Full-screen window manager.
 @end direntry
 
-@c For examples, use a literal escape in info.
-@ifinfo
-@set esc 
-@end ifinfo
-@iftex
-@set esc <ESC>
-@end iftex
+@c Display Esc as ^[ no matter the medium.
+@set esc ^[
 
 @ifinfo
 This file documents the @code{Screen} virtual terminal manager.
@@ -78,7 +73,8 @@ by the Foundation.
 @ifinfo
 @top Screen
 This file documents the @code{Screen} virtual terminal manager, version
-@value{version}.
+@value{version}.  The MirOS specific changes are, however, documented in the
+manual page instead of here.  See screen(1) for details.
 @end ifinfo
 
 @menu
@@ -2888,7 +2884,7 @@ h=^B:l=^F:$=^E} would set some keys to b
 users.
 If your terminal sends characters, that cause you to abort copy mode,
 then this command may help by binding these characters to do nothing.
-The no-op character is `@' and is used like this: @code{markkeys @@=L=H}
+The no-op character is @samp{@@} and is used like this: @code{markkeys @@=L=H}
 if you do not want to use the `H' or `L' commands any longer. 
 As shown in this example, multiple keys can be assigned to one function
 in a single statement.
